cypress/support/commands.js

```diff
@@ -22,4 +22,18 @@
 //
 //
 // -- This will overwrite an existing command --
-// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
\ No newline at end of file
+// Cypress.Commands.overwrite('visit', (originalFn, url, options) => { ... })
+
+
+// add a command to login that uses a session, so the user will remain logged in throughout the test vs. needing to log in before each example.
```

Comment:
I see that you added `cy.login` to the requests test for a logged in user. does this comment mean that after that login, the user is still logged in for subsequent tests in that file? or that the user should already be logged in when the file is loaded?

Reply:
yes - cy.login creates a "session" so if we wanted to keep the user logged in throughout the file, it would need to be called after line 1 vs. inside of the example.  but since the first test is with a logged out user, i put it directly in the example in this case.

Reply:
ah. per file then. thanks.

Reply:
```suggestion
// add a command to login that uses a session, so the user will remain logged in throughout the test file vs. needing to log in before each example.
```